### PR TITLE
Resolve gateway

### PR DESCRIPTION
**Pull request by VadimKhoroshiltsev**

### CODE REVIEW

File: packages/resolve-gateway/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# `resolve-query`
```
Comment (IhostVlad) [inline packages/resolve-gateway/README.md]: Implicit copy-paste from `resolve-query`, should be removed or replaced to appropriate readme.

File: packages/resolve-gateway/package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  },
+  "author": "reimagined team",
+  "license": "MIT",
+  "keywords": [
+    "cqrs",
+    "eventsourcing",
+    "es",
+    "ddd"
+  ],
+  "homepage": "https://github.com/reimagined/resolve/tree/master/packages/resolve-query#readme",
```
Comment (IhostVlad) [inline packages/resolve-gateway/package.json]: Query => gateway

File: benchmarks/benchmark-base/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "benchmark-base",
-  "version": "0.0.6",
+  "version": "0.0.6-alpha.6924b406",
```
Comment (IhostVlad) [inline benchmarks/benchmark-base/package.json]: Version changed to alpha - possible wrong `lerna` affected

File: packages/resolve-gateway/src/index.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+            eventStore,
+            gateway
+        });
+        return result;
+    }, {});
+
+    return async (gatewayName) => {
+        const executor = executors[gatewayName.toLowerCase()];
+
+        if (executor === undefined) {
```
Comment (ralkov) [inline packages/resolve-gateway/src/index.js]: === is redudant

**Pull request closed without merging**